source/adapters/level_zero/event.cpp

```diff
@@ -866,8 +866,11 @@ UR_APIEXPORT ur_result_t UR_APICALL urEventRelease(
     ur_event_handle_t Event ///< [in] handle of the event object
 ) {
   Event->RefCountExternal--;
```

Comment:
get return value from this atomic and use it for comparison, no need to do atomic twice